docs/contrib/keycloak.md

```diff
@@ -0,0 +1,148 @@
+# Keycloak Identity Provider for REMS
+
+Using docker, docker-compose, and Keycloak, you can roll your own local IdP for use with REMS.
+
+## Quick-ish Start
+
+1. **Setting up the Docker files**
+    1. Create a `Dockerfile` for your keycloak. To start, it can be as simple as the following:
+    ```
+    FROM quay.io/keycloak/keycloak:11.0.2
+    ```
+    2. Create a `docker-compose.yaml` file for REMS and your Keycloak IdP. Mount your custom REMS config to `/rems/config/config.edn`. Example:
+    ```
+    version: "3.7"
+    services:
+    rems:
+        mem_reservation: 200m
+        mem_limit: 500m
+        ports:
+        - "127.0.0.1:3001:3000"
+        volumes:
+        - ./services/rems/simple-config.edn:/rems/config/config.edn
+        depends_on:
+        - db
+        - keycloak
+    db:
+        image: postgres:9.6
+        environment:
+        POSTGRES_USER: rems
+        POSTGRES_PASSWORD: remspassword
+        mem_reservation: 30m
+        mem_limit: 150m
+        ports:
+        - "127.0.0.1:5432:5432"
+    keycloak:
+        environment:
```

Comment:
Here I used the image from the repo so `image: jboss/keycloak:latest`. Perhaps we need to add it here too?

Reply:
Sure thing, you mean to get rid of the keycloak `Dockerfile` and just pull the image with docker-compose, right? I'm fine with either option.

Reply:
If it's possible to do this without building your own Keycloak then that is simpler, right? At least I didn't need to but I'm not so familiar with the whole setup. Do you need to do some things there (e.g. the ones I did with the `hosts`-file?)